openstack/baremetal/httpbasic/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Package httpbasic provides support for http_basic bare metal endpoints.
+
+Example of obtaining and using a client:
+
+        client, err := httpbasic.NewBareMetalHttpBasic(httpbasic.Endpoints{
```

Comment:
nit: spacing issue

Reply:
by spacing issue you mean the 
```
	IronicEndpoing: "http://localhost:6385/v1/",
	IronicUser: "myUser",
	IronicUserPassword: "myPassword",
```
and should be:
```
	IronicEndpoing:         "http://localhost:6385/v1/",
	IronicUser:                 "myUser",
  	IronicUserPassword: "myPassword",
```
thanks

Reply:
No, if you look at the alignment of `client` and the closing `})` a few lines below, they don't match.